main.js

```diff
@@ -5,4 +5,81 @@ const navToggle = () => {
+    '<img src="images/modinosdesk.png" alt="Offenbach Grand Prix Homepage">',
+
+  ],
+  text: [
+    'A fully online opera-singing competition, where the audience can comment and applaud, and where the jury comments in full view of the audience.  Automated application process, which includes video and biog upload, as well as an automatically generated contract in the end.',
+
+    'Spoiled Eggs was the original template for 2 projects and those two projects were used to promote Spoiled Eggs.  It is a review site for opera, where people can rage click "applause" or "throwing eggs" and of course comment.',
+
+    'A site meant as an extension of Opera Classica Europa`s Inthega classifieds, which is a platform intended on facilitating touring in Germany.',
+
+    "The advertisement and application process for Opera Classica Europa's Opera Studio program, called the Opera Pathway.",
+
+    'A template intended at hosting music competitions, mostly those that were dedicated to a composer.  The Offenbach GP was such a competition later in time.',
+
+    'The advertisement and application process to enter the Callas Prize competition, which was supposed to take place at the Alte Oper Frankfurt for the finals.  Unfortunately cancelled due to COVID.',
+
+    "My personal singing portfolio and also the first site I ever coded.  In it's initial form it was merely a list, but later I created an interesting, albeit somewhat glitchy, navbar system.",
+
+  ],
+
+  liveButton: [
+    '<a href="https://www.offenbachgp.com/" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="https://spoiledeggs.eu5.org/" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="#" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="#" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="#" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="#" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="https://philipmodinos.ueuo.com" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+
+  ],
+
+  sourceButton: [
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+  ],
+
+};
```

Comment:
Hi, the previous reviewer is correct to mention that this is not the requested way to implement this project. please kindly make a change to follow this format:
- [ ] have each item in its own object and the objects would be inside an array. that way you can loop through the array of objects implemented in the HTML for each one. This is much more scalable and maintainable and is a much better solution than what you currently have
![image](https://user-images.githubusercontent.com/18730106/151249355-99163836-2241-4a60-a8a4-453ae6f776c8.png)